Src/FluentAssertions/Common/TypeMemberReflector.cs

```diff
@@ -40,16 +40,13 @@ private static List<PropertyInfo> GetPropertiesFromHierarchy(Type typeToReflect,
         bool includeExplicitlyImplemented = memberVisibility.HasFlag(MemberVisibility.ExplicitlyImplemented);
 
         return GetMembersFromHierarchy(typeToReflect, type =>
-        {
-            return
-                from p in type.GetProperties(AllInstanceMembersFlag | BindingFlags.DeclaredOnly)
-                where p.GetMethod is { } getMethod
-                    && (IsPublic(getMethod) || (includeExplicitlyImplemented && IsExplicitlyImplemented(getMethod)))
-                    && (includeInternal || !IsInternal(getMethod))
-                    && !p.IsIndexer()
-                orderby IsExplicitImplementation(p)
-                select p;
-        });
+            from p in type.GetProperties(AllInstanceMembersFlag | BindingFlags.DeclaredOnly)
```

Comment:
🤔 I don't think this is better. I would like to keep that decision on a case by case basis.